### DISCUSSION TITLE
Gmsl camera and antmedia server.

### DISCUSSION THREAD

**Discussion by SauravTirkey** [ant-media/Ant-Media-Server#5308] [Q&A]
Can we use Gmsl camera with antmedia server? Is it possible for Gmsl camera to pull live video stream with antmedia server? If yes, then what is the procedure?

**Comment by Mohit-3196** [comment]
Hi @SauravTirkey,
How are you?

As far as I know, AMS does not support GMSL cameras natively because GMSL cameras typically transmit video using standard streaming protocols such as MIPI CSI-2 (Camera Serial Interface) or FPD-Link III.
In a typical setup, the GMSL camera's video data can be captured by an interface or processing unit and then the captured video can be encoded and streamed using RTMP or as a stream source to AMS.

Thank you

**Reply by Mohit-3196** [reply]
Hi @SauravTirkey ,
I hope you are doing well.

I was wondering if my answer could help with your question. Please let me know if you have any questions or if I can do anything for you.

Thank you again.
